pepper-apis/src/main/java/org/broadinstitute/ddp/cache/ModelChangeType.java

```diff
@@ -0,0 +1,5 @@
+package org.broadinstitute.ddp.cache;
+
+public enum ModelChangeType {
+    USER
+}
```

Comment:
I am anticipating there will be a couple of other types added, as discussed in PR intro. The type paired with an identifier will let us know what data will need to be evicted from cache.